generator/src/main/java/com/scottlogic/deg/generator/reducer/ConstraintMapper.java

```diff
@@ -18,20 +18,20 @@
 import java.util.stream.Stream;
 import java.util.stream.StreamSupport;
 
-public class ConstraintReducer {
+public class ConstraintMapper {
```

Comment:
would it be worth adding some documentation about what the class does, to avoid any confusion in future?

Reply:
is that adequate, or would more information be helpful?

Reply:
The explains what it does.
if you start the comment with `/**` then the [javadoc tool](https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format) will pick it up.